test/proto/bookstore.proto

```diff
@@ -0,0 +1,159 @@
+syntax = "proto3";
+
+package bookstore;
```

Comment:
This is already a simplified version compared to what we have in Cloud Endpoints, but feel free to add an item to #1192 to see what can be done here.